tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,48 +87,69 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val wskpropsBackup = TestUtils.backupWskProps()
```

Comment:
Yeah, I agree that the tests should leave no footprints in the user wsk CLI configuration.  I like the cli argument that specifies the property file.  With the decision on external configuration file handling outstanding, I'll switch the tests back to using the existing env variable